docs/ardana/rpm-file-checks.md

```diff
@@ -0,0 +1,59 @@
+# rpm file checks
+
+[`ardana-job`](testing.md) performs some automated checks to ensure
+that Ardana doesn't mess with the filesystem inappropriately by
+removing or modifying rpm-owned files, or adding files into /usr.
+This is done by running `rpm -Va` and `rpm -qf` before and after
+Ardana deployment (including a tempest run, if deployment succeeded),
+and comparing the results.  If any tampering is discovered, `init.yml`
+will fail with a helpful error.
+
+[Whitelists](../../scripts/jenkins/ardana/ansible/files/) are used to
+ignore known exceptions, some of which need to be fixed in the future.
```

Comment:
We have more files, like rpm-verify-modification-fixmes.txt, is this also a whitelist? do we need to document this and how is maintained?

Reply:
Renamed to `rpm-verify-modification-whitelist.txt` and documented.